### PR TITLE
Feature/sink multiple input schemas

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java [hunk 1]
```diff
@@ -252,9 +262,9 @@ private void validateSchema(Class<? extends Driver> jdbcDriverClass, String tabl
          ResultSet tables = connection.getMetaData().getTables(null, null, tableName, null)) {
 
       Preconditions.checkArgument(tables.next(), "Table %s does not exist. " +
```
Comment (jayfjin) [inline database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: can we move the string onto a new line so we don't have this odd indentation

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java [hunk 2]
```diff
@@ -302,7 +312,7 @@ private void validateFields(Schema inputSchema, ResultSet rs) throws SQLExceptio
     }
 
     Preconditions.checkArgument(invalidFields.isEmpty(), "Couldn't find matching database column(s) " +
```
Comment (jayfjin) [inline database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: same here

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java [hunk 3]
```diff
@@ -107,30 +109,37 @@ public void prepareRun(BatchSinkContext context) {
...
+      dbSinkConfig, connectionString, dbColumns, driverClass)));
   }
 
-  private void setColumnsInfo(List<Schema.Field> fields) {
+  /**
+   * Extracts column info from input schema. Later it is used for metadata retrieval
+   * and insert during query generation. Override this method if you need to escape column names
+   * for databases with case-sensitive identifiers
+   *
+   * @param fields input schema fields
```
Comment (jayfjin) [inline database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: remove, this param field isn't helpful

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java (hunk 1 above)
Comment (ibarbashov) [reply database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: done

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java (hunk 2 above)
Comment (ibarbashov) [reply database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: done

File: database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java (hunk 3 above)
Comment (ibarbashov) [reply database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java]: done

**Pull request merged**